spec/unit/sidekiq/retryable_jobs_filter_spec.rb

```diff
@@ -32,5 +32,19 @@ def build_notice(job = nil)
       filter.call(notice)
       expect(build_notice).to_not be_ignored
     end
+
+    context 'with max_retries specified' do
+      it "ignores notices below max_retries" do
+        notice = build_notice('retry' => 5, 'retry_count' => 1, 'max_retries' => 2)
+        filter.call(notice)
+        expect(build_notice).to_not be_ignored
```

Comment:
According to the spec description this has to be `expect(build_notice).to be_ignored` (unless I am missing something).

Reply:
Oh, I just noticed this wasn't actually fixed. Should still be `expect(notice).to be_ignored` according to the spec description. P.S. Also note that it's `build_notice` instead of `notice`. We basically introduce the same bug as on line 22 where you fixed it 😉